src/services/pallets/PalletsForeignAssetsService.ts

```diff
@@ -56,10 +57,23 @@ export class PalletsForeignAssetsService extends AbstractService {
 			if (foreignAssetData) {
 				// remove any commas from multilocation key values e.g. Parachain: 2,125 -> Parachain: 2125
 				const foreignAssetMultiLocationStr = JSON.stringify(foreignAssetData[0]).replace(/(\d),/g, '$1');
-				const foreignAssetMultiLocation = api.registry.createType(
-					'XcmV3MultiLocation',
-					JSON.parse(foreignAssetMultiLocationStr),
-				);
+				const XcmV3MultiLocation = ['StagingXcmV3MultiLocation', 'XcmV3MultiLocation'];
```

Comment:
We can get rid of all of this, the try catches, and the xcm multilocation array, and just pass in an object of the multilocation. Given this [comment](https://github.com/polkadot-js/api/issues/5741#issuecomment-1772151935) we dont want to be using createType directly anymore unless its testing, and we want to avoid using a nested try catch at all costs.